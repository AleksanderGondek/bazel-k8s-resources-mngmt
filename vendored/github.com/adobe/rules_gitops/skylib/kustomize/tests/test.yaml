# namespace should be replaced
apiVersion: v1
kind: Service
metadata:
  name: akube-dns
  namespace: kube-system
  labels:
    k8s-app: kube-dns
spec:
  selector:
    k8s-app: kube-dns
  ports:
  - name: dns
    port: 53
---
#namespace should be set
apiVersion: v1
kind: Service
metadata:
  name: bkube-dns
  labels:
    k8s-app: kube-dns
spec:
  selector:
    k8s-app: kube-dns
  ports:
  - name: dns
    port: 53
---
# should be NO namespace since this object is globally scoped
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: kubedobe:cluster-admin
subjects:
- kind: Group
  name: adcloud_k8s_sudo
  apiGroup: rbac.authorization.k8s.io
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: cluster-admin
