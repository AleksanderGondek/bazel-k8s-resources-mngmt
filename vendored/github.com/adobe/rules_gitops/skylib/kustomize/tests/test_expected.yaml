apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: kubedobe:cluster-admin
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: cluster-admin
subjects:
- apiGroup: rbac.authorization.k8s.io
  kind: Group
  name: adcloud_k8s_sudo
---
apiVersion: v1
kind: Service
metadata:
  labels:
    k8s-app: kube-dns
  name: akube-dns
  namespace: newnamespace-1
spec:
  ports:
  - name: dns
    port: 53
  selector:
    k8s-app: kube-dns
---
apiVersion: v1
kind: Service
metadata:
  labels:
    k8s-app: kube-dns
  name: bkube-dns
  namespace: newnamespace-1
spec:
  ports:
  - name: dns
    port: 53
  selector:
    k8s-app: kube-dns
